game-core/src/main/java/games/strategy/triplea/delegate/AbstractPlaceDelegate.java

```diff
@@ -540,7 +541,7 @@ private String moveAirOntoNewCarriers(
     }
     final List<Unit> fighters = producer.getUnitCollection().getMatches(ownedFighters);
     final Collection<Unit> movedFighters =
-        getRemotePlayer().getNumberOfFightersToMoveToNewCarrier(fighters, producer);
+        bridge.getRemotePlayer().getNumberOfFightersToMoveToNewCarrier(fighters, producer);
```

Comment:
Are you sure this is safe? Just checking as I've ran across examples where the local data was not the same as the data on the bridge. It manifested as unexpected null data as the bridge data was only partially maintained.

Reply:
100% sure.
I noticed I haven't actually created seperate commits for this, but I basically inlined the getRemotePlayer() method call using IntelliJ refactoring